test/actions/linkActions.test.js

```diff
@@ -0,0 +1,39 @@
+import test from 'blue-tape'
+import * as actions from '../../src/actions/linkActions'
+
+const CALENDAR = {type: 'course', id: 'MI-RUB', date: '2015-10-11'}
+
+test('calendarUrl()', t => {
+  t.test('calendarUrl() with all parameters', st => {
+    st.plan(1)
```

Comment:
I’m just curious, why did you use `plan()` instead of `end()` at the bottom of the test function as in other tests?